### PR TITLE
Add GEM space

### PR DESCRIPTION
**Pull request by onebyteman**

### CODE REVIEW

File: spaces/gem/skin.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.sushi {
```
Comment (e-asia) [inline spaces/gem/skin.scss:1]: The css selector should be identical to the space key.
```suggestion
.gem {
```

Comment (YumingHe) [reply spaces/gem/skin.scss]: addressed

File: spaces/gem/index.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "key": "gem",
+  "symbol": "GEM",
+  "name": "Gem",
+  "defaultView": "core",
+  "address": "0x90f62B96a62801488b151fF3c65eaC5Fae21a962",
+  "token": "0x90f62B96a62801488b151fF3c65eaC5Fae21a962",
+  "core": [
+    "0x28f97Cd17F09F2e93d46cc18a035a714dE605Fe5",
```
Comment (bonustrack) [inline spaces/gem/index.json:9]:
```suggestion
    "0x28f97Cd17F09F2e93d46cc18a035a714dE605Fe5"
```

Comment (YumingHe) [reply spaces/gem/index.json]: addressed

**Review by e-asia — APPROVED** [review]

**Comment by YumingHe** [comment]
could we merge this?

**Pull request merged**